src/imitation/policies/exploration_wrapper.py

```diff
@@ -53,9 +55,14 @@ def __init__(
         # Choose the initial policy at random
         self._switch()
 
-    def _random_policy(self, obs: np.ndarray) -> np.ndarray:
+    def _random_policy(
+        self,
+        obs: np.ndarray,
+        state: Optional[Tuple[np.ndarray, ...]],
+        episode_start: Optional[np.ndarray],
+    ) -> Tuple[np.ndarray, Optional[Tuple[np.ndarray, ...]]]:
         acts = [self.venv.action_space.sample() for _ in range(len(obs))]
-        return np.stack(acts, axis=0)
+        return np.stack(acts, axis=0), None  # is this right?
```

Comment:
This feels like a weird case. If you have a stateful policy, it'll randomly lose all state every time exploration happens. On the other hand, if you just pass the state through this policy, it'll behave strangely since the world state will have changed without the hidden state having a chance to catch up. I think None is an okay solution - or, we could throw or warn if the state passed to the random policy is not None to indicate that the behavior is undefined?

Reply:
Or we could simply always pass an empty state to the wrapped policy

Reply:
Stateful polices and exploration don't seem like they go together very well. I'm struggling to think of a single example of an off policy stateful algorithm that is used in practice. This should definitely have some kind of warning, maybe even an error. I would bump this one up to @AdamGleave.

Reply:
Updated to throw. (Talked with @AdamGleave offline and he agrees)